src/SystemBenchmark.jl

```diff
@@ -37,6 +38,12 @@ function getsysteminfo()
     push!(df, ["info","WORD_SIZE","",getinfofield(systeminfo, "WORD_SIZE:")])
     push!(df, ["info","LIBM","",getinfofield(systeminfo, "LIBM:")])
     push!(df, ["info","LLVM","",getinfofield(systeminfo, "LLVM:")])
+    @static if VERSION ≥ v"1.7-DEV"
+        push!(df, ["info","BLAS","",first(splitext(basename(first(BLAS.get_config().loaded_libs).libname)))])
+    else
+        push!(df, ["info","BLAS","",missing])
```

Comment:
Do you know ways to get the implementation of BLAS with Julia v1.6?

Reply:
I don't. Perhaps that can be fixed in a follow-on PR. This PR seems beneficial to merge now